### ISSUE TITLE
Add bminer to miners list

### ISSUE THREAD

**Issue by bbgobie** [MultiPoolMiner/MultiPoolMiner#989]
Would love to have this one added as it seems to possibly be the fastest for equihash right now.
https://bitcointalk.org/index.php?topic=2519271.0

**Comment by UselessGuru** [comment]
Coming.

BTW: It is not really faster :-(

**Comment by UselessGuru** [comment]
Bminer cannot be started using the methods available in the present code (same issue as eminer). Miner window opens and closes immediately. Running the miner with the same command line directly from a dos / powershell session runs OK!

**Comment by alanblythe** [comment]
I just downloaded your Bminer.zip and it's running fine. I have 6 x 1070s, running 2.7.1.4. I had modified the script to open the windows as Normal rather than Minimized. This miner seems much more dependent on CPU and I'm also getting 10% less performance than DSTM and EWBF

**Comment by cacipriani** [comment]
I got it working. I am not a coder and do not know Powershell, but hacked together an API and Miner file.
This will only work if you have one GPU, if someone more well-versed in Powershell wants to work on the API to support summing the hashrate of multiple GPUs be my guest :) I pieced it together based on how the other API/Miner files are written.

First, unzip BMiner into .\Bin\Bminer\
Place BMINER.ps1 into the API folder (.\API\)
Place BMiner.txt into the Miners folder

Run your start.bat and it should benchmark and work.
[BMiner.zip](https://github.com/MultiPoolMiner/MultiPoolMiner/files/1628205/BMiner.zip)
